.dockerignore

```diff
@@ -0,0 +1,5 @@
+npm-debug.log
```

Comment:
Each `dockerfile` folder setup should have it's `.dockerignore` that way, we can easily tell it's scope

Reply:
@andela-sjames just a little explanation on this; I had to take the .dockerignore back into the root directory because it must be in the root directory of the build context.

Reply:
Do you have a reference to this explanation? Docs maybe.

Reply:
Where is the dockefile using this as its build context?

Reply:
```
Before the docker CLI sends the context to the docker daemon, 
it looks for a file named .dockerignore in the root directory of the context.
```
So who or what is referencing this context ?

Reply:
I found this too https://medium.com/@LihauTan/took-me-hours-to-realise-why-docker-build-ignores-my-dockerignore-and-this-is-what-ive-learned-2f87c770ea9c